### PR TITLE
Adding t.Helper calls to Pointers & errors section

### PR DESCRIPTION
**Pull request by midnightrun**
Hi,
I found that we are not using `t.Helper` in this section and was confused, if this was done by purpose or or not.

If latter is true, this is a PR to fix it :)

Chéers,
Camal

### CODE REVIEW

**Comment by quii** [comment]
Sorry for the confusion! It was not on purpose and thanks for taking the time to fix it

File: pointers-and-errors.md [hunk 1]
```diff
@@ -310,7 +310,8 @@ There's some duplication in our tests, lets refactor that out.
 func TestWallet(t *testing.T) {
 
     assertBalance := func(t *testing.T, wallet Wallet, want Bitcoin) {
-        got := wallet.Balance()
+        t.Helper()
+    	got := wallet.Balance()
```
Comment (pityonline) [inline pointers-and-errors.md]: @midnightrun Looks like an indent mixed TAB and spaces. Could you keep the same indents with the context?

Comment (midnightrun) [reply pointers-and-errors.md]: I used `gofmt` for formatting. Didn't change any default here. But looks strange to me too :/

Comment (pityonline) [reply pointers-and-errors.md]: Me either. Because gofmt only format go codes but not markdown.

Comment (midnightrun) [reply pointers-and-errors.md]: Ok, will fix it with another PR. Thanks @pityonline for making me aware :)

**Pull request merged**